lib/signaling/participant.js

```diff
@@ -152,6 +156,31 @@ class ParticipantSignaling extends StateMachine {
     }
   }
 
+
+  /**
+   * @param {TrackSid} trackSid
+   * @param {Priority?} subscriberPriority
+   * @returns {void}
+   */
+  updateTrackSubscriberPriority(trackSid, subscriberPriority) {
+    if (this._trackPrioritySignaling) {
```

Comment:
I am also bit concerned about a possible race condition - if user happen to set track priority while MSP object was null, because it was not yet negotiated or because VMS was transitioning from failover. One solution is to always queue the update priority request behind `trackPriorityPromise`. But that adds bit of complications. BTW I think  same potential problem exists in https://github.com/twilio/twilio-video.js/pull/733 as well.